From 1085e32a9ddc232963512923332094a58a05d1af Mon Sep 17 00:00:00 2001
From: "Edgar E. Iglesias" <edgar.iglesias@xilinx.com>
Date: Wed, 14 Dec 2016 09:34:58 -0800
Subject: [PATCH] zynqmp-arm: Explicitly instantiate the Cortex-A53s

Explicitly instantiate the Cortex-A53s. This avoids issues with
inconsistent default settings for the default ARMv8 core in QEMU.

Signed-off-by: Edgar E. Iglesias <edgar.iglesias@xilinx.com>
Reviewed-by: Alistair Francis <alistair.francis@xilinx.com>
---
 zynqmp-arm.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zynqmp-arm.dtsi b/zynqmp-arm.dtsi
index 837631b..887705d 100644
--- a/zynqmp-arm.dtsi
+++ b/zynqmp-arm.dtsi
@@ -30,7 +30,7 @@
 #include "zynqmp.dtsh"
 
 #define APU_COMMON(n,p)																			\
-	compatible = "arm,armv8";																	\
+	compatible = "cortex-a53-arm-cpu";																	\
 	d-cache-line-size = <0x20>;																	\
 	d-cache-size = <0x8000>;																	\
 	device_type = "cpu";																		\
